front/app/api/files/types.ts

```diff
@@ -85,7 +85,10 @@ export interface IFileData {
   type: string;
   attributes: IFileAttributes;
   relationships: {
+    preview?: { data: IRelationship | null };
     uploader: { data: IRelationship };
+    project?: { data: IRelationship };
+    attachments?: { data: IRelationship[] };
```

Comment:
I built this off of your latest API branch for the file attachments, but there weren't any attachments in the Relationships for the files? :thinking:  So I wasn't able to implement a solution which actually linked to or had the titles of the resources.